static_frame/core/store.py

```diff
@@ -167,9 +169,9 @@ def get_field_names_and_dtypes(*,
 
             # add index names in front of column names
             if include_columns:
-                field_names.extend(columns_values)
+                field_names.extend(columns_values) # pyright: ignore
```

Comment:
Do you know why this is ignored? `field_names: tp.Sequence[TLabel]` and `columns_values: tp.Sequence[TLabel]`

Reply:
I only had the fortitude for very limited exploration of such questions: the overall typing maintenance was so high on-boarding pyright I erred on the side of ignoring!